src/renderer/extensions/vueNodes/composables/useNodePointerInteractions.ts

```diff
@@ -10,29 +12,51 @@ import { useNodeDrag } from '@/renderer/extensions/vueNodes/layout/useNodeDrag'
 import { isMultiSelectKey } from '@/renderer/extensions/vueNodes/utils/selectionUtils'
 import { app } from '@/scripts/app'
 
-// Forward keydown events to litegraph's processKey when Vue nodes have focus
-let keydownForwardingInitialized = false
-
-function initKeydownForwarding() {
-  if (keydownForwardingInitialized) return
-  keydownForwardingInitialized = true
-
-  document.addEventListener(
-    'keydown',
-    (e) => {
-      const canvas = app.canvas
-      if (!canvas) return
-      if (e.target === canvas.canvas) return
-      canvas.processKey(e)
-    },
-    true
-  )
+// Forward spacebar key events to litegraph for panning when Vue nodes have focus
+let spacebarForwardingInitialized = false
+
+function isEditableElement(el: Element | null): boolean {
+  if (!el) return false
+  const tag = el.tagName.toUpperCase()
+  if (tag === 'INPUT' || tag === 'TEXTAREA') return true
+  if (el instanceof HTMLElement && el.isContentEditable) return true
+  return false
+}
+
+function initSpacebarForwarding() {
+  if (spacebarForwardingInitialized) return
+  spacebarForwardingInitialized = true
+
+  const { space } = useMagicKeys()
+  const activeElement = useActiveElement()
```

Comment:
I don't think we need to (or should) have these instantiated conditionally. We can just include them in the composable directly.